Makefile

```diff
@@ -1,4 +1,4 @@
-VERSION := 11.0.1
+VERSION := 12.0.0
```

Comment:
Not sure this is necessary? Or if we do change, could just be 11.0.2 since no functional changes?

Reply:
Ok!